src/extensions/cite-apa/index.test.ts

```diff
@@ -0,0 +1,19 @@
+import { select, text, whenReady } from '../../util'
+
+test('DOM manipulations', async () => {
+  document.body.innerHTML = `
```

Comment:
There're two different styles of this assignment across the test files:
```JavaScript
document.body.innerHTML = ...
```
and 
```JavaScript
  const body = document.body
  body.innerHTML = ...
```
Might be worth picking one?

Reply:
Yep, fair point, will go with the former.

Reply:
Actually, after looking at the code, went for the latter.